### PR TITLE
Added mpl-scatter-density to list of tools

### LINKED ISSUE #42: Mention mpl-scatter-density in large data section?
The [mpl-scatter-density](https://github.com/astrofrog/mpl-scatter-density) is a lightweight package that makes it possible to show large (up to ~10^9 points) datasets with Matplotlib. It might be worth mentioning it in the 'Large-data rendering' section but the issue is that that section specifically mentions 'server-side' rendering. Can we relax this wording to allow tools like mpl-scatter-density to be mentioned?

### PR DESCRIPTION
**Pull request by astrofrog**
Fixes #42

### CODE REVIEW

File: tools/tools.yml [hunk 1]
```diff
@@ -248,7 +248,7 @@
       badges: pypi, travis, azure, coveralls, conda, site
 
 - name: Large-data rendering
-  intro: Tools for server-side rasterizing/aggregating data before visualization.
+  intro: Tools for rasterizing/aggregating data before visualization, in some cases server-side.
```
Comment (jbednar) [inline tools/tools.yml]:
```suggestion
  intro: Tools for rasterizing/aggregating data before visualization, to allow  in some cases server-side.
```

File: tools/tools.yml [hunk 2]
```diff
@@ -248,7 +248,7 @@
       badges: pypi, travis, azure, coveralls, conda, site
 
 - name: Large-data rendering
-  intro: Tools for server-side rasterizing/aggregating data before visualization.
+  intro: Tools for rasterizing/aggregating data before visualization, to allow  in some cases server-side.
```
Comment (jbednar) [inline tools/tools.yml]: Sorry; somehow I must have submitted a version in progress, not the final one:

```suggestion
  intro: Tools for rasterizing/aggregating data before visualization, which can make rendering faster, allow larger datasets, or make it feasible to use remote datasets (with server-side rendering).
```

**Comment by astrofrog** [comment]
@jbednar - fixed!

**Comment by jbednar** [comment]
Thanks for the contribution!

**Pull request merged**